helm/kubeblocks/falkordb-cluster-omnistrate/tests/e2e_omnistrate/test_omnistrate_replication.py

```
@@ -0,0 +1,476 @@
+
+    def test_replication_stop_start(self, instance):
+        """
+        Test 3: Verify instance can be stopped and started.
+        
+        SKIPPED: Omnistrate API does not yet support stop/start operations.
+        This test will be added once API support is available.
+        """
+        pytest.skip("Stop/start not supported by Omnistrate API yet")
+        
+        # Add data before stop
+        add_data(instance, ssl, key="test_stopstart", n=200, network_type=network_type)
```

Ruff (0.14.10)

Undefined name `ssl`

(F821)